### PR TITLE
Wifi 5ghz support

### PR DESCRIPTION
**Pull request by salvatore-coppola**

### CODE REVIEW

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiAccessPoint.java [hunk 1]
```diff
@@ -47,6 +47,15 @@
     public long getFrequency();
 
     /**
+     * The Wi-Fi channel number in use by the access point.
+     * 
+     * 
+     */
```
Comment (MMaiero) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiAccessPoint.java:56]: I believe a since is needed here

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiAccessPoint.java [hunk 2]
```diff
@@ -47,6 +47,15 @@
...
+    public int getChannel();
+
+    /**
+     * 
+     * /**
```
Comment (MMaiero) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiAccessPoint.java]: Probably unwanted /**

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiConfig.java [hunk 3]
```diff
@@ -224,7 +224,7 @@ public String getWifiCountryCode() {
      * Set the Wifi Country Code
      * 
      * @param wifiCountryCode
-     * @since 2.2
+     * @since 1.1.0
```
Comment (MMaiero) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiConfig.java]: Why this change?

**Review by pierantoniomerlino — APPROVED** [review]

**Review by pierantoniomerlino — APPROVED** [review]

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiConfig.java [hunk 4]
```diff
@@ -234,7 +234,7 @@ public void setWifiCountryCode(String wifiCountryCode) {
      * Get the list of Wifi channels and frequencies
      * 
      * @return List of wifi channels and frequencies
-     * @since 2.2
+     * @since 1.1.0
```
Comment (MMaiero) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiConfig.java]: Still wrong version here

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiConfig.java [hunk 5]
```diff
@@ -243,7 +243,7 @@ public void setWifiCountryCode(String wifiCountryCode) {
     /**
      * Set the list of channel frequencies
      * 
-     * @since 2.2
+     * @since 1.1.0
```
Comment (MMaiero) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiConfig.java]: And here

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiConfig.java (hunk 4 above)
Comment (salvatore-coppola) [reply kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiConfig.java]: rectified

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiConfig.java (hunk 5 above)
Comment (salvatore-coppola) [reply kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiConfig.java]: rectified

**Review by MMaiero — APPROVED** [review]
